views/idea.handlebars

```diff
@@ -125,15 +125,34 @@
                     <span class="d-block mt-2">
                         {{name}}
                     </span>
+
+                    {{#if ../is_owner}}
+                    <div class="position-absolute bottom-0 start-50 translate-middle-x" style="z-index:100">
+                         <button
+                            class='mb-3 btn btn-danger delete-resource'
+                            value='{{id}}'
+                            data-id='{{id}}'
+                        >Delete</button>
+                   
+                        {{#unless (is_match type 'image')}}
+                        <button
+                            class='mb-3 btn btn-success'
+                            value='{{id}}'
+                            type="{{type}}"
```

Comment:
I think you want a different html attribute here. The button type attribute is usually something like 'button' or 'submit': https://www.w3schools.com/tags/att_button_type.asp